imagestreams/jupyter-cpp-cling/imagestream.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: jupyter-cpp-cling
+  namespace: redhat-ods-applications
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: >-
+      https://github.com/nerc-images/jupyter-cpp-cling
+    opendatahub.io/notebook-image-name: >-
+      Jupyter C++ Cling
+    opendatahub.io/notebook-image-desc: >-
+      An OpenShift AI Image running Jupyter Lab for C++ development.
+      - Based on the [Xeus Cling project](https://github.com/jupyter-xeus/xeus-cling)
+      on GitHub for Jupyter Lab Notebook integration.
+      - Uses C++ Kernel 11, 14, and 17 and gcc-c++ compiler support.
+      - Used by Professor Leonidas Kontothanassis, MassMutual Professor of the
+      Practice of Computing & Data Sciences + Director of Industry Engagement.
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    - name: latest
+      annotations: null
+      from:
+        kind: DockerImage
+        name: 'quay.io/nerc-images/jupyter-cpp-cling:latest'
```

Comment:
Are we confident about using the `latest` tag here (and elsewhere)? This can cause problems when an update to `latest` introduces an incompatible change and breaks everything that depends on it.

Reply:
@larsks I can switch it to `main`, but it's basically the same thing because latest is based off of the main branch. Is it better to use a specific sha, or create tags and releases, I could do that too?

Reply:
Using the commit hash in addition to updating `latest` is probably a good place to start, since it allows images to be pinned to a known working version. If it makes since to produce versioned releases, we should also do that, and tag the images accordingly.

Reply:
@larsks I have updated my 3 image streams to use a `1.0.0` tagged release I created for each image repository with the current README for each as the release description. We can still use a sha tag, but I prefer tag version for more clarity for updated versions.

Reply:
Oh, I prefer tagged versions as well, but that takes a little more discipline to set up and I wasn't sure how much time you wanted to spend on this :).